### PR TITLE
Add width and offset options to cnx

### LINKED ISSUE #58: Allow to specify the width and offset of the bar
Hey ! Nice project, just getting started with it!

I have this use case, with 2 monitors, where I want a bar on each monitor.

Currently, it just stretches across the 2 monitors. 
I have a small patch that add 2 `Option`s to the `Bar::new` , namely `width` and `x_offset`. 
I feel it's just a quickhack :sweat_smile: 

`height` is not useful AFAIK, and `y_offset` makes not much sense (well it could maybe, not sure). 

I was thinking of maybe adding some `with_x_offset`, `with_width` to the `Bar` struct, wdyt?

### PR DESCRIPTION
**Pull request by Sh4d1**
Fixes #58 

Signed-off-by: Patrik Cyvoct <patrik@ptrk.io>

### CODE REVIEW

File: cnx/src/lib.rs [hunk 1]
```diff
@@ -141,8 +141,16 @@ pub use bar::Position;
 /// [`add_widget()`]: #method.add_widget
 /// [`run()`]: #method.run
 pub struct Cnx {
+    /// The position of the Cnx bar
     position: Position,
+    /// The list of widgets attached to the Cnx bar
     widgets: Vec<Box<dyn Widget>>,
+    /// The (x,y) offset of the bar
+    /// It can be used in order to run multiple bars in a multi-monitor setup
+    offset: (i16, i16),
```
Comment (psibi) [inline cnx/src/lib.rs:150]: Instead of tuple can you make this as Struct instead ? Also, can you make it as optional too.

**Review by psibi — CHANGES_REQUESTED** [review]
Thanks, LGTM apart from a minor change that is required.

File: cnx/src/lib.rs (hunk 1 above)
Comment (Sh4d1) [reply cnx/src/lib.rs]: Added the struct, though it seems more logical to use the fefault of 0,0, rather than making it an Option

Comment (psibi) [reply cnx/src/lib.rs]: Yeah, that sounds better.

**Review by psibi — APPROVED** [review]
LGTM. Can you also update the changelog ?

**Comment by psibi** [comment]
@Sh4d1 Sorry, I guess you need to resolve the conflicts.

**Comment by Sh4d1** [comment]
Done !

**Comment by psibi** [comment]
Thank you!

**Pull request merged**